rfcs/theming-color-scheme-strategy.md

```diff
@@ -0,0 +1,168 @@
+
+While developing or modifying an Orbit component, it is important to use alias variables whenever possible. Remember that aliases are theme sensitive, which is what we aim for in order to support a dark theme as well as keeping apps concise.
+
+Here instead of using `color: var(--o-ui-global-marine-900);` we refer to the defined primary text color.
+
+``` css
+// Label.css
+.o-ui-field-label {
+    color: var(--o-ui-alias-text-1);
+    white-space: nowrap;
+}
+```
+
+If you think it is absolutely necessary to add a new alias, this need to be discussed with the Orbit core team. Once approved this alias need to be added as a symbol in Figma, and documented in Orbit.
+
+#### Global variables
+
+Global variables have their use, spacing, typography and other static values are not tied to semantic variables, aka themes are not affecting these. 
+
+```css
+// Label.css
+.o-ui-field-label-required {
+    margin-left: var(--o-ui-global-scale-alpha);
+}
+```
+
+#### Local variables
+
+Sometimes a component needs reusable values through it's UI while not explicitly needing to be shared between other components. It is alright to create reusable local variables in a component's CSS file. These variables should be prefixed with the components name and be still be accesible globally and be scoped to the `o-ui` class. Here is a typical scenario:
+
+```css
+// Input.css
+.o-ui {
+    --o-ui-input-border-color: var(--o-ui-alias-border-1);
+    --o-ui-input-border-radius: var(--o-ui-rounded-border-radius);
+}
+
+.o-ui-input {
+    color: var(--o-ui-input-border-color);
+    // ...
+}
+```
+
+As you can see the local variables are not prefixed by global or alias while keeping a logical scope through it's `-{c omtonent-name} prefix. A subcomponent would have access to it, e.g. NumberInput. Defining these variables in the component's CSS file makes it easier to remove unnecessary CSS when removing a component.
+
+#### Utility variables
+
+Utility variables are variables that helps crafting experiences, while being utilitarian. These are not values that have been discussed with designers, a circle being a circle there is no art direction on these. Creating a circle with a border-radius is a common CSS trick that is used all over Orbit and the web in general. Instead of repeating the same line of CSS all the time, there is an utility for that. Many more utilities are available through Orbit, check Orbit's documentation for more information.
+
+```css
+.o-ui {
+  --o-ui-circular-border-radius: 100%;
+}
+
+// ...
+
+.circle {
+    border-radius: var(--o-ui-circular-border-radius);
+}
+```
+
+### Implementation in Orbit
+
+The *Theme Provider* componen is responsible of adding the 3 needed classes: `-o-ui o-ui-{theme} -o-ui-{color-scheme}) in order to benefit from the complete Orbit UI experience.
+
+Orbit's utility variables are declared at the root CSS file `react-components/src/index.css` as well as generic CSS declarations like the app default font-color.
```

Comment:
Should mention that utility variables shouldn't have any prefix (if this is what we choose to do)